examples/canvas/canvas.go

```diff
@@ -56,7 +57,11 @@ func loop() {
 func main() {
 	wnd := g.NewMasterWindow("Canvas", 600, 600, g.MasterWindowFlagsNotResizable, nil)
 
-	img, _ := g.LoadImage("gopher.png")
+	img, err := g.LoadImage("gopher.png")
+	if err != nil {
+		log.Fatal(err)
+	}
+
```

Comment:
@AllenDang @ianling what do you think about upgrading project to go 1.16 and using go:embed here?

Reply:
We could add an example using go:embed. What's the nil-pointer bug?

Reply:
when I'm working in local giu fork, I was running `go run examples/canvas/canvas.go`.
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x575973]

goroutine 1 [running, locked to thread]:
github.com/AllenDang/giu/imgui.(*OpenGL3).createImageTexture(0xc00006e230, 0x0, 0xc00015e100, 0x44160000, 0xc00010de10)
	/home/user/git/d2/me/giu/imgui/RendererOpenGL3.go:440 +0x113
github.com/AllenDang/giu/imgui.(*OpenGL3).LoadImage(0xc00006e230, 0x0, 0xc000110360, 0x487635, 0xc0000221e0)
	/home/user/git/d2/me/giu/imgui/RendererOpenGL3.go:417 +0x35
github.com/AllenDang/giu.NewTextureFromRgba.func1(0x57fdc7, 0xc0000221e0)
	/home/user/git/d2/me/giu/Texture.go:27 +0x42
github.com/faiface/mainthread.CallVal.func1()
	/home/user/go/pkg/mod/github.com/faiface/mainthread@v0.0.0-20171120011319-8b78f0a41ae3/mainthread.go:84 +0x32
github.com/faiface/mainthread.Run(0xc00000e5b8)
	/home/user/go/pkg/mod/github.com/faiface/mainthread@v0.0.0-20171120011319-8b78f0a41ae3/mainthread.go:44 +0xb1
github.com/AllenDang/giu.(*MasterWindow).Run(0xc00015d420, 0x8b7738)
	/home/user/git/d2/me/giu/MasterWindow.go:250 +0x65
main.main()
	/home/user/git/d2/me/giu/examples/canvas/canvas.go:64 +0xaf
exit status 2
```
It was caused by missing image file in ./gopher.png (it is ./examples/canvas/gopher.png)
In ghis example, image could by embeded by go:embed and it'll fix bug, I suppose

Reply:
Don't run the example via ` go run examples/canvas/canvas.go`, use `cd examples/canvas && go run .` instead. Because in the demo code the image is referenced by using the relative path like './gopher.png'.

Reply:
oh, ok, I can see, thanx